<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="9" column="0">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="maximumSize">
       <size>
        <width>900</width>
        <height>840</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>URW Gothic [urw]</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="5" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <family>URW Gothic [urw]</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Прибыль за указанный период</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QDateEdit" name="dateEdit">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2024</year>
          <month>7</month>
          <day>5</day>
         </datetime>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2024</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="maximumDate">
         <date>
          <year>2024</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="font">
         <font>
          <family>URW Gothic [urw]</family>
         </font>
        </property>
        <property name="text">
         <string>Популярность композиций</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="status">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>100</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>URW Gothic [urw]</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>customer</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>URW Gothic [urw]</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>       до</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QDateEdit" name="dateEdit_2">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2024</year>
          <month>7</month>
          <day>5</day>
         </datetime>
        </property>
        <property name="maximumDate">
         <date>
          <year>2024</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
        <property name="minimumDate">
         <date>
          <year>2024</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="name">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>1920</width>
          <height>1080</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>URW Gothic [urw]</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Profile: </string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="pushButton_3">
        <property name="font">
         <font>
          <family>URW Gothic [urw]</family>
         </font>
        </property>
        <property name="text">
         <string>Кол-во использованых цветов</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QComboBox" name="comboBox">
        <property name="font">
         <font>
          <family>URW Gothic [urw]</family>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Insert</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Новый заказ</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QComboBox" name="comboBox_2">
        <property name="font">
         <font>
          <family>URW Gothic [urw]</family>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Delete</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Заказ</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="pushButton_4">
        <property name="font">
         <font>
          <family>URW Gothic [urw]</family>
         </font>
        </property>
        <property name="text">
         <string>Все заказы на эту дату</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic [urw]</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Цветы</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Композиции</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Все заказы</string>
    </property>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Мои заказы</string>
    </property>
    <addaction name="action_4"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Показать цветы</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Показать композиции</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Показать все заказы</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Показать мои заказы</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
